src/openloco/graphics/gfx.cpp

```diff
@@ -214,8 +215,7 @@ namespace openloco::gfx
         {
             // Removed detection of offscreen-drawing
 
-            uint8_t chr = *str;
```

Comment:
We can't do this yet. Existing savegames will have different encodings, as well as object file strings.